packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -66,7 +71,8 @@ class BottomAppBar extends StatefulWidget {
 
   /// The bottom app bar's background color.
   ///
-  /// When null defaults to [ThemeData.bottomAppBarColor].
+  /// When null defaults to [ThemeData.bottomAppBarTheme.color], if that's null,
```

Comment:
If this property is null then [ThemeData...] is used. If that's null then  ...

Same comment for the other properties

Reply:
Done.